nav2_bringup/launch/nav2_params.yaml

```diff
@@ -1,8 +1,59 @@
-map_server:
+amcl:
   ros__parameters:
-    yaml_filename: "test_map.yaml"
-DwbController:
+    use_sim_time: True
```

Comment:
I thought you were going to put the `use_sim_time` parameters in a seperate YAML file so we could choose whether to include it or not?

Reply:
Wasn't planning to do that as part of this PR. We can do that. I'll add it to the list of TODO items for the integration branch once we get these changes in.